vyked/bus.py

```diff
@@ -231,13 +220,11 @@ def _handle_log_change(self, packet, protocol):
 
 
 class PubSubBus:
-    PUBSUB_DELAY = 5
 
     def __init__(self, registry_client, ssl_context=None):
         self._pubsub_handler = None
         self._registry_client = registry_client
         self._clients = None
-        self._pending_publishes = {}
         self._ssl_context = ssl_context
 
     def create_pubsub_handler(self, host, port):
```

Comment:
def register_for_subscription(self, host, port, node_id, clients):
            self._clients = clients
            subs_list = []
            xsubs_list4registry = []
            xsubs_list4redis = []
            for client in filter(lambda x: isinstance(x, TCPServiceClient), clients):
                for fn in filter(lambda x: callable(x), dir(client)):

                    if getattr(fn, 'is_subscribe', False):
                        subs_list.append(self._get_pubsub_key(client.name, client.version, fn.__name__))
                    elif getattr(fn, 'is_xsubscribe', False):
                        xsubs_list4registry.append((client.name, client.version, fn.__name__, getattr(fn, 'strategy')))
                        xsubs_list4redis.append(self._get_pubsub_key(client.name, client.version, fn.__name__,
                                                                     node_id=node_id))

            self._registry_client.x_subscribe(host, port, node_id, xsubs_list4registry)
            yield from self._pubsub_handler.subscribe(subs_list + xsubs_list4redis, handler=self.subscription_handler)